compatibility/bazel_tools/testing.bzl

```
@@ -244,6 +244,42 @@ excluded_test_tool_tests = [
             },
         ],
     },
+    {
+        "start": "1.0.0",
+        "platform_ranges": [
+            {
+                "end": "0.0.0",
+                "exclusions": [
+                    "CommandServiceIT:CSCreateAndBadExerciseChoice",
+                    "CommandSubmissionCompletionIT:CSCRefuseBadChoice",
+                ],
+            },
+        ],
+    },
+    {
+        "end": "1.14.0-snapshot.20210602.7086.0.f36f556b",
+        "platform_ranges": [
+            {
+                "start": "0.0.0",
+                "exclusions": [
+                    "CommandServiceIT:CSCreateAndBadExerciseChoice",
+                    "CommandSubmissionCompletionIT:CSCRefuseBadChoice",
+                ],
+            },
+        ],
+    },
+    {
+        "start": "0.0.0",
```

```suggestion
        "start": "1.14.0-snapshot.20210602.7086.1.f36f556b",
```
Note the `.1.` this is effectively an exclusive bound on the snapshot so we don’t have to update it next week.